src/bci_build/package/python.py

```diff
@@ -56,7 +57,13 @@ def _get_python_kwargs(py3_ver: _PYTHON_VERSIONS, os_version: OsVersion):
             "PATH": "$PATH:/root/.local/bin",
             "PIP_VERSION": pip3_replacement,
         },
-        "package_list": [f"{py3}-devel", py3, pip3, "curl", "git-core"]
+        "package_list": [
+            Package(py3),
+            Package(pip3),
+            f"{py3}-devel",
+            "curl",
+            "git-core",
+        ]
```

Comment:
Wrong type, `package_list` is `list[str]|list[Package]` not `list[str|Package]`. We can change the type, but that will require further code adjustments.

Reply:
Yes, true, I missed that.

My idea was to only allow `list[Package]`, but this would affect all images.
It has its benefits on the long run if we go this way.